File: src/main/java/com/tobe/healthy/lessonhistory/repository/LessonHistoryFilesRepository.java
package com.tobe.healthy.lessonhistory.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.tobe.healthy.lessonhistory.domain.LessonHistoryFiles;

@Repository
public interface LessonHistoryFilesRepository extends JpaRepository<LessonHistoryFiles, Long> {
}

File: src/main/kotlin/com/tobe/healthy/lessonHistory/repository/LessonHistoryFilesRepository.kt
package com.tobe.healthy.lessonHistory.repository

import com.tobe.healthy.lessonHistory.domain.entity.LessonHistoryFiles
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface LessonHistoryFilesRepository : JpaRepository<LessonHistoryFiles, Long>
